### DISCUSSION TITLE
how to Detection output format

### DISCUSSION THREAD

**Discussion by Liripo** [quarto-dev/quarto-cli#10900] [Q&A]
### Description

```qmd
format: 
  test1-html: 
    format-links: false
  test2-html:
    output-file: report-test2.html
```
If it is a different html template, is it possible for me to determine the `output format` of my current document?
`knitr::is_html_output` can only determine whether it is html.

**Comment by mcanouil** [comment]
Could you elaborate on your use case?

You can "detect" the format via a Lua filter.
